src/main/kotlin/fi/metatavu/muisti/settings/MqttSettings.kt

```diff
@@ -10,8 +10,9 @@ package fi.metatavu.muisti.settings
  * @author Antti Leppä
  */
 data class MqttSettings (
```

Comment:
Is this data class actually useful anymore? You could pass the settings MqttConnection.connect without it as well

Reply:
settings are also used for keeping the topic etc to be reused at MqttConnection, so I'd rather keep it